src/test/java/org/skife/jdbi/v2/sqlobject/TestGetGeneratedKeysPostgres.java

```diff
@@ -67,9 +72,10 @@ public Object withHandle(Handle handle) throws Exception
         public String findNameById(@Bind long id);
     }
 
-    @Ignore
     @Test
     public void testFoo() throws Exception {
+        assumeTrue(postgresIsInstalled);
```

Comment:
Hm, the JUnit Wiki that I linked above says
> A failing assumption in a @Before or @BeforeClass method will have the same effect as a failing assumption in each @Test method of the class.
which I think means it should work?

(I voted the question up, wow that does not deserve a -3!)